cabal.project

```diff
@@ -12,14 +12,10 @@ repository cardano-haskell-packages
 
 -- See CONTRIBUTING for some Nix commands you will need to run if you
 -- update either of these.
--- NOTE: repeating the index-state for Hackage is a temporary workaround
--- for hackage.nix limited parsing ability.
--- Bump both the following dates if you need newer packages from Hackage
-index-state: 2023-04-28T00:00:00Z
 index-state:
   , hackage.haskell.org 2023-04-28T00:00:00Z
 -- Bump this if you need newer packages from CHaP
-  , cardano-haskell-packages 2023-04-27T14:03:58Z
+  , cardano-haskell-packages 2023-05-09T09:56:38Z
```

Comment:
Updating `index-state` in ledger requires an update to sources. See [Updating dependencies](https://github.com/input-output-hk/cardano-ledger/blob/master/CONTRIBUTING.md#updating-dependencies) section of `CONTRIBUTING.md` doc
```shell
$ niv update cardano-haskell-packages
```

Reply:
I think this really needs to be a CI test. I do not use Nix, and I could easily get it wrong.

Reply:
> I think this really needs to be a CI test

It is, except nix (cicero/hydra) CI has been quite unreliable, so that's why they aren't required. Hopefully the state of affairs will change in the future, in which case failure to updated flakes will be rejected by CI.

Reply:
Done